org.scala-ide.sdt.core/src/org/scalaide/ui/editor/ReconcilingStrategy.scala

```diff
@@ -25,7 +30,7 @@ class ReconcilingStrategy(sourceEditor: SourceCodeEditor, documentListener: IDoc
   override def reconcile(partition: IRegion) {
     for (doc <- document) {
       val errors = sourceEditor.getInteractiveCompilationUnit.forceReconcile()
-      sourceEditor.updateErrorAnnotations(errors)
+      sourceEditor.updateErrorAnnotations(errors, null)
```

Comment:
Good point. I'l name it. It's the JDT JavaElement corresponding to this source, and I forgot exactly when it's used. In this case, there is no such element, but the API expects it anyway.